cluster-maintenance/cleanup_unused_pv_pvc.sh

```diff
@@ -0,0 +1,43 @@
+function output_help {
+    echo "Usage: `basename $0` <kubernetes_namespace_to_clean>";
+}
+
+if [ "$#" -ne 1 ]; then
+    echo "Illegal number of parameters, you need to specify k8s namespace"
+    echo ""
+    output_help
+    exit 1
+fi
+
+NAMESPACE=$1
+
+CLAIMS_IN_USE=$(kubectl get pods -n $NAMESPACE -o=json | jq -c '.items[] | {claimName: .spec.volumes[] | select( has ("persistentVolumeClaim") ).persistentVolumeClaim.claimName }.claimName')
+ALL_CLAIMS=$(kubectl get pvc -n $NAMESPACE | tail -n +2 | awk '{print $1}')
+
+echo "--------- CLAIMS IN USE ---------"
+echo $CLAIMS_IN_USE
+
+echo "> Going to delete claims"
+for claim_to_delete in $ALL_CLAIMS
+do
+	to_delete=true
+    for claim_in_use in $CLAIMS_IN_USE
+	do
+		if [[ "\""$claim_to_delete"\"" == $claim_in_use ]]
+		then
+			echo "Skipping claim "$claim_to_delete
+	    	to_delete=false
+    	fi 
+	done
```

Comment:
github shows some confusing indentation here - I think the accepted convention is space characters 

while the nested for loop works, might be nicer to have a function to check - see options here: https://stackoverflow.com/questions/8063228/how-do-i-check-if-a-variable-exists-in-a-list-in-bash